python-checks/src/test/resources/checks/numpyIsNan.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+
+def foo(x):
+    if x == np.nan: print(1)  # Noncompliant
+    #  ^^^^^^^^^^^
+    if np.nan == x: print(1)  # Noncompliant
```

Comment:
Looks really good, great job! 

I was thinking, maybe we could add a small test with for example:

```
import numpy

def foo():
    if x == numpy.nan
``` 
Or instead of `import numpy as np` `import numpy as something_else`. 
WDYT?

Reply:
Thanks for the tip, and for raising the issue. Turns out that my check didn't work is cases where we do `from numpy import nan`. I have now addressed the change in `b80fd05f99ef4c801e5dbdb6b3be08d39b8966b5`.